src/RedirectionForm.php

```diff
@@ -174,6 +174,16 @@ public function render() : string
         return call_user_func_array($renderer, $data);
     }
 
+    /**
+     * Returns the target URL.
+     *
+     * @return string target URL
+     */
+    public function getTargetUrl(): string
```

Comment:
why do we need this method? you can run `toJson` method and retrieve redirection form in JSON format.
**as you know we sometimes we redirect to bank page using POST method, so getTargetUrl is not suitable here.** 
it's better to **get JSON format and send the suitable POST or GET request according to it**.

Reply:
you could use use `render` method to generate redirection form (i think you use laravel)

```php
// in laravel controller, the below code renders a redirection form and pass it as Laravel's view
return $purchasedPayment->pay()->render();

```